Src/Public/Invoke-AsBuiltReport.Rubrik.CDM.ps1

```diff
@@ -433,7 +433,7 @@ function Invoke-AsBuiltReport.Rubrik.CDM {
                                 @{N = "Username"; E = { $_.username } },
                                 @{Name = "VM Linking"; Expression = { $_.conflictResolutionAuthz } },
                                 @{Name = "Certificate"; E = { $_.caCerts } }
-                                Write-Verbose -Message "[Rubrik] [$($brik)] [Cluster Settings] Retrieving vCloud Directory Information"
+                                Write-Verbose -Message "[Rubrik] [$($brik)] [Cluster Settings] Retrieving vCloud Director Information"
```

Comment:
I'd like to switch `Write-Verbose` to [`Write-PScriboMessage`](https://github.com/iainbrighton/PScribo/blob/dev/Src/Public/Write-PScriboMessage.ps1) to keep the verbose output format consistent.

Reply:
And done!